extensions/analyticsdx-vscode-templates/test/vscode-integration/schemas/templateInfoSchema.test.ts

```diff
@@ -296,44 +309,55 @@ describe('template-info-schema.json hookup', () => {
       { jsonpath: ['externalFiles', 0, 'condition'], completions: ['""', '${...} expression'] },
       { jsonpath: ['externalFiles', 0, 'rows'], completions: ['5'] },
       { jsonpath: ['externalFiles', 0, 'overwriteOnUpgrade'], completions: ['${...} expression'] },
+      { jsonpath: ['externalFiles', 0, 'onFailure', 'defaultStatus'], completions: ['"Fail"', '"Skip"', '"Warn"'] },
```

Comment:
This tests that the expected code completions are showing up in vscode based on the json schema (e.g. enum values, examples, and snippets).